packages/backend/src/accounting/psql/ledger-account/index.ts

```diff
@@ -32,18 +32,20 @@ export async function createAccount(
 
 export async function getLiquidityAccount(
   deps: ServiceDependencies,
-  accountRef: string
+  accountRef: string,
+  trx?: TransactionOrKnex
 ): Promise<LedgerAccount | undefined> {
-  return LedgerAccount.query(deps.knex)
```

Comment:
Curious what the intent of adding a transaction here is, since these look like they're just reading the database.

Reply:
If we are creating the account and reading it in the same transaction, we also need it here: https://github.com/interledger/rafiki/pull/1882/files#diff-4b287690c751dd38900f4202d32f2e0c023b378614a7363fbf791226fa926f45R152-R167